### DISCUSSION TITLE
[css fit-content] chrome 104 and latest firefox

### DISCUSSION THREAD

**Discussion by prgrmrwy** [DouyinFE/semi-design#1034] [Q&A] [answered]
https://github.com/DouyinFE/semi-design/blob/f76bd5879dc8289c6ff30dac9e717a0f8e9d1cb6/packages/semi-foundation/_portal/portal.scss#L15

new chromium(version 104) has corrected available-size behaviour, when in usecase, fit-content will trigger unexpected behaviour,
maybe semi can update to fit it

chromium developer anwsers: https://github.com/chromium/chromium/commit/9bd53baedd0ca471cfe188becd4abc417c245f78#r80842895

**Comment by pointhalo** [comment, marked as answer]
seems related to this issue https://github.com/DouyinFE/semi-design/issues/1022.
We have updated this css configuration in the beta version, the stable version will be released on Friday.
